code/AssetLib/FBX/FBXDocument.h

```diff
@@ -55,9 +55,13 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define _AI_CONCAT(a,b)  a ## b
 #define  AI_CONCAT(a,b)  _AI_CONCAT(a,b)
 
+/* Use an 'illegal' default FOV value to detect if the FBX camera has set the FOV. */
```

Comment:
Please use // style comments. This will make it much easier to put bigger blocks into comments.